src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -33,11 +33,12 @@ file that was distributed with this source code.
 {% endapply %}
 {% endblock %}
 
-{% block form_help %}
-    {% apply spaceless %}
-        <span class="help-block sonata-ba-field-widget-help sonata-ba-field-help">{{ parent() }}</span>
-    {% endapply %}
-{% endblock %}
+{% block form_help -%}
+    {% if help is not empty %}
+        {% set help_attr = help_attr|merge({class: (help_attr.class|default('') ~ ' help-block sonata-ba-field-help')}) %}
```

Comment:
Why is `sonata-ba-field-widget-help` removed?

Reply:
No usages was found by me (:

Reply:
It can theorically be used by a developer for specific css or js. We shouldnt remove class IMHO.

Reply:
4 classes for just help is too many...
You should understand that  in theorically they can relate on this element as:
```CSS
.help-block p {
 margin-top: 3px;
}
```
But after this fix need: 
```CSS
.help-block {
 margin-top: 3px;
}
```

Reply:
Maybe but it's still a Bc break.
The only thing you can do would be to deprecate some css class on 3.x and remove then in master.

Reply:
Done